src/shared/copy/notifications.ts

```diff
@@ -1096,6 +1096,18 @@ export const communityTemplateRenameFailed = (): Notification => ({
   message: `We've successfully installed your template but weren't able to name it properly. It may appear as a blank template.`,
 })
 
+export const editCheckCodeWarning = (): Notification => ({
+  ...defaultErrorNotification,
+  style: NotificationStyle.Info,
+  message: `Changes to Check code may prevent you from editing the Check in the visual editing experience.`,
```

Comment:
small thing when writing, but ends up actually saving time when reading and maintaining code:

the backtick
```
`this is my ${variable}`
```

implies it's going to use [string templating](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) - which is the only time you'd use the backtick string character. if you're not doing string templating, just use a single / double quote instead.